backend/conferences/models/keynote.py

```diff
@@ -0,0 +1,82 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from model_utils.models import TimeStampedModel
+from ordered_model.models import OrderedModel
+
+from pycon.constants import COLORS
+
+
+class Keynote(OrderedModel, TimeStampedModel):
+    conference = models.ForeignKey(
+        "conferences.Conference",
+        on_delete=models.CASCADE,
+        verbose_name=_("conference"),
+        related_name="keynotes",
+        null=False,
+    )
+    slug = models.SlugField(_("slug"), max_length=200)
+    title = models.CharField(
```

Comment:
The text fields shoudn't be `I18nCharField` ? How we are gonna translate the keynotes pages?

Reply:
I don't think we should translate them. Having them in Italian might confuse making people think that it will be in Italian

Reply:
it's fine in both ways for me. But maybe in the frontend I will add a little flag or something to emphasize again the language... what do you think?

Reply:
I added a "Language" on the right-side box :)